packages/procaptcha-react/src/components/LoadingSpinner.tsx

```diff
@@ -0,0 +1,35 @@
+/** @jsx jsx */
+/** @jsxImportSource @emotion/react */
+import { darkTheme, lightTheme } from './theme.js'
+import { useMemo } from 'react'
+import styled from '@emotion/styled'
+
+export interface LoadingSpinnerComponentProps {
+    themeColor: 'light' | 'dark'
+}
+
+export const LoadingSpinner = ({ themeColor }: LoadingSpinnerComponentProps) => {
```

Comment:
Should have aria labels on stuff like this. Not super important right now since we don't have accessibility options impl right now, but at the least is SEO friendly for the dapps hosting us

Reply:
Sure, will add them.

Reply:
I've added the aria labels to the checkbox.